recipes/server.rb

```diff
@@ -29,6 +30,20 @@
   notifies :restart, "service[#{node['nfs']['service']['server']}]"
 end
 
+# RHEL7 has some extra requriements per
+# https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/Storage_Administration_Guide/nfs-serverconfig.html#s2-nfs-nfs-firewall-config
+if rhel? && node['platform_version'].to_f >= 7.0
+  include_recipe 'sysctl::default'
```

Comment:
@atomic-penguin When I run a kitchen converge on CentOS 7.1, it both executes the sysctl and sets up the service. Are you seeing different behavior? As best I can tell from looking at the definition of `include_recipe`, it adds the recipe to the node's run_context which behaves as if it had been included in the run_list.

Older versions of sysctl did not require any recipe inclusion and just worked with resources. They made a breaking change without following semantic versioning and required a recipe to be included (which broke things for people). I think the referenced issue is in relation to that change, and does not indicate that the recipe has to be defined in the run_list. Since sysctl_apply runs an execute block that notifies a ruby block defined in `sysctl::default`, it just has to be included sometime in the compile phase of the recipe evaluation for the target to exist.